db/import-tmpl/mob_item_ratio.yml

```diff
@@ -0,0 +1,71 @@
+# Specific Item Drop Ratio Database
+# This file is a part of rAthena.
+#   Copyright(C) 2021 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Overrides for global item_rate* values from conf/battle/drops.conf
+###########################################################################
+#
+# Settings
+#
+###########################################################################
+# - Item          Aegis item name.
+#   Ratio         Drop rate.
+#   List:         List of monsters affected (optional, default are all monsters).
+#     - Mob       Monster Aegis name (unique to the List).
+#       Clear     (optional) Remove the Monster. Monster Aegis name required.
+###########################################################################
+#
+# Additional description
+#
+###########################################################################
+#
+# ItemID base drop rates defined in mob_db will not get multiplied by global item_rate* values (aka drop rates) from
+# conf/battle/drops.conf. Instead Ratio will be used (100 = 1x).
+# If no MonsterID is specified, all monsters will be affected, otherwise only listed ones.
```

Comment:
```suggestion
# Item base drop rates defined in mob_db.yml will not get multiplied by global item_rate* values (aka drop rates) from
# conf/battle/drops.conf. Instead, Ratio will be used (100 = 1x).
# If no Mob is specified, all monsters will be affected, otherwise only the ones listed.
```